_extras/prereqs.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Prerequisites
+permalink: /prereqs/
+---
+
+This lesson assumes you have a working knowledge of Python and some previous exposure to the Bash shell. 
+These requirements can be fulfilled by:  
+a) completing a Software Carpentry Python workshop **or**  
+b) completing a Data Carpentry Ecology workshop (with Python) **and** a Data Carpentry Genomics workshop **or**  
+c) coursework in or independent learning of both Python and the Bash shell.
+
+### Bash shell skills
+
+The skill set listed below is covered in any Software Carpentry workshop, as well
+as in Data Carpentry's Genomics workshop. These skills can also be learned
+through coursework or independent learning. 
+
+Be able to: 
+* Identify and navigate to your home directory.
+* Identify your current working directory.
+* Navigating directories using `pwd`, `ls`, `cd <subdirectory>`, and `cd ..`
+* Run a Python script from the command line.
+
+### Python skills
+
+This skill set listed below is covered in both Software Carpentry's Python workshop and
+in Data Carpentry's Ecology workshop with Python. These skills can also be learned
+through coursework or independent learning. 
+
+Be able to: 
+* Use the assignment operator to create int, float, and string variables.
```

Comment:
would highlight those types as code, also exchanged `string` with `str` (to match `int` vs `integer`).

```suggestion
* Use the assignment operator to create `int`, `float`, and `str` variables.
```